hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapService.java

```diff
@@ -338,4 +339,16 @@ public void rollbackTransaction(String transactionId) {
     public void dispatchEvent(EventData event, EntryListener listener) {
         dispatcher.dispatchEvent(event, listener);
     }
+
+    @Override
+    public Map<String, LocalMultiMapStats> getStats() {
+        Map<String, LocalMultiMapStats> mapStats = new HashMap<String, LocalMultiMapStats>();
+        for (int i = 0; i < nodeEngine.getPartitionService().getPartitionCount(); i++) {
```

Comment:
I think looping multimap partitions is not the right way. As far as I see, you only multimap names. And there's also a `statsMap` in `MultiMapService` which holds local statistics.